packages/platform-ios/src/link/getHeaderSearchPath.js

```diff
@@ -53,8 +53,8 @@ export default function getHeaderSearchPath(sourceDir, headers) {
   const directories = union(headers.map(path.dirname));
 
   return directories.length === 1
-    ? `"$(SRCROOT)${path.sep}${path.relative(sourceDir, directories[0])}"`
-    : `"$(SRCROOT)${path.sep}${path.relative(
+    ? `"$(SRCROOT)/${path.relative(sourceDir, directories[0])}"`
```

Comment:
Won't `path.relative` use the OS specific separator?

Reply:
I think you can use `path.posix` to switch to the posix implementation regardless of the OS.

Reply:
@qwertzguy https://github.com/react-native-community/cli/pull/635